textalyze.js

```
@@ -24,6 +24,15 @@ const sanitize = str => typeof str == 'string'
   ? str.toLowerCase()
   : '';
 
+const itemFrequency = array => {  
+  if (typeof array != 'object' && !array.isArray)
```

I'm not sure this is the right behavior here.  I would just let the thing throw an error rather than make the error go away.

---

You mean something like this?

```javascript
if (typeof array !== 'object' && !array.isArray) {
    throw new Error('This function does not accept non-array values');
}
```